examples/combobox/aria1.1pattern/listbox-combo.html

```diff
@@ -264,7 +264,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
               </ul>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-escape">
```

Comment:
This is same ID as line 222, but won't tests be different?

Reply:
Changing this to: `listbox-key-escape` and the other to `textbox-key-escape`